core/src/main/java/org/openstack4j/api/OSClient.java

```diff
@@ -208,6 +216,14 @@
     TelemetryService telemetry();
 
     /**
+     * Returns the Gnocchi Telemetry Service API
+     *
+     * @return the Gnocchi telemetry service
+     */
+    TelemetryGnocchiService gnocchiTelemetry();
```

Comment:
Is this the Gnocchi Telemetry Service a separate OpenStack service that should live here or is it be part of the telemetry service (e.g. just different TSDB/backend to ceilometer)?